data-service/src/main/java/uk/gov/gchq/palisade/service/data/service/SimpleDataService.java

```diff
@@ -59,43 +58,59 @@ public SimpleDataService(
         this.dataReader = dataReader;
     }
 
-    public CompletableFuture<Optional<DataReaderRequest>> authoriseRequest(final DataRequest dataRequest) {
+    /**
+     * Query for the references.  It will return the information needed to retrieve the resources.  If there is no
+     * data to be returned, a {@link ForbiddenException} is thrown.
+     *
+     * @param dataRequest data provided by the client for requesting the resource
+     * @return reference to the resources that are to be returned to client
+     * @throws ForbiddenException if there is no authorised data for the request
+     */
+    public CompletableFuture<AuthorisedDataRequest> authoriseRequest(final DataRequest dataRequest) {
         LOGGER.debug("Querying persistence for token {} and resource {}", dataRequest.getToken(), dataRequest.getLeafResourceId());
-        CompletableFuture<Optional<AuthorisedRequestEntity>> futureRequestEntity = this.persistenceLayer.getAsync(dataRequest.getToken(), dataRequest.getLeafResourceId());
+        CompletableFuture<Optional<AuthorisedRequestEntity>> futureRequestEntity = persistenceLayer.getAsync(dataRequest.getToken(), dataRequest.getLeafResourceId());
         return futureRequestEntity.thenApply(maybeEntity -> maybeEntity.map(
-                entity -> new DataReaderRequest()
-                        .context(entity.getContext())
-                        .user(entity.getUser())
-                        .resource(entity.getLeafResource())
-                        .rules(entity.getRules())
-                )
+                entity -> AuthorisedDataRequest.Builder.create()
+                        .withResource(entity.getLeafResource())
```

Comment:
again, change ordering to follow other similar templates

Reply:
See comment for AuditErrorMessage